meeteval/wer/__main__.py

```diff
@@ -115,14 +115,12 @@ def _load_hypothesis(hypothesis: 'list[Path]'):
             return CTMGroup.load(hypothesis).to_stm()
         else:
             return CTMGroup.load([hypothesis]).to_stm()
-    elif filename.endswith('.stm'):
-        return STM.load(hypothesis)
     elif filename.startswith('/dev/fd/') or filename.startswith('/proc/self/fd/'):
         # This is a pipe, i.e. python -m ... <(cat ...)
         # For now, assume it is an STM file
         return STM.load(hypothesis)
```

Comment:
Add a ToDo to support more formats?

Reply:
Where exactly would you add it and which formats would you support in the future?

Reply:
Something like that:
```
# For now, assume it is an STM file
# ToDo: Support more formats that can be clearly distinguished, just from the content.
return STM.load(hypothesis)
```

Reply:
Done